tests/unit_tests/bal/test_transforms.py

```
@@ -0,0 +1,39 @@
+
+    g = jnp.stack([x * p[0] for p in params_ensemble])
+    expected = jnp.sqrt(1 / 5) * jnp.sum(g, axis=0)
+    assert jnp.allclose(result, expected)
+
+
+def test_batch_features():
+    key = jax.random.PRNGKey(0)
+    params = jnp.zeros(5)  # dummy
+    x_batch = jax.random.normal(key, (4, 3))  # batch, features
+
+    def mock_feature_fn(params, x):
```

Ruff (0.15.1)

[warning] Unused function argument: `params`

(ARG001)